platforms/gke/base/use-cases/inference-ref-arch/terraform/teardown.sh

```diff
@@ -21,45 +21,47 @@ start_timestamp=$(date +%s)
 export ACP_TEARDOWN_CORE_PLATFORM=${ACP_TEARDOWN_CORE_PLATFORM:-"true"}
 
 export TF_VAR_initialize_backend_use_case_name="inference-ref-arch/terraform"
-export TF_VAR_resource_name_prefix="inf"
 
```

Comment:
Why was the prefix removed?

Reply:
Leaving this open while we settle this down in the other thread.